src/Adapter/Module/ModulePresenter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Module;
+
+use Currency;
+use PrestaShop\PrestaShop\Adapter\Product\PriceFormatter;
+use PrestaShop\PrestaShop\Core\Foundation\Templating\PresenterInterface;
+
+class ModulePresenter implements PresenterInterface
+{
+    /**
+     * @var Currency
+     */
+    private $currency;
+
+    /** @var PriceFormatter */
+    private $priceFormatter;
+
+    public function __construct(Currency $currency)
+    {
+        $this->currency = $currency;
+        // Not declared as a Symfony service :(
```

Comment:
Will be done for the ModulePresenter, but the other presenters will be declared by the guy who created them ;)